pool/pool_test.go

```diff
@@ -661,29 +680,29 @@ func TestWaitPresence(t *testing.T) {
 }
 
 func TestCopySessionTokenWithoutSignatureAndContext(t *testing.T) {
```

Comment:
This function was removed, so you can drop this test, I guess.

Reply:
25ac588b11cf56898c0ad2064bf02e350e717d82